browser-test/test.endtoend.ts

```diff
@@ -0,0 +1,144 @@
+/**
```

Comment:
While running these tests, I've received warnings (during webpack) that say this file and `test.grpc-fallback.ts` exceed the recommended size limit of 244KiB and this might impact web performance. Is that something we should try to avoid or is it fine as is?

Reply:
It's fine since it's just about tests, but if you'd like to fix it, we can add `performance: {hints: false},` to webpack config that's read in `karma.conf.js`.

```
const webpackConfig = require('./webpack.config.js');
webpackConfig.performance = { hints: false };
```

should just work - can you check it? :)

Reply:
I'm not sure this is a big deal because these tests shouldn't be shipped with the production packed module. So it only really has a chance of effecting performance during our testing which isn't a big deal.

Reply:
Yep, looks like it works!